### ISSUE TITLE
What's the Windows 11 story?

### ISSUE THREAD

**Issue by zevisert** [AudioBand/AudioBand#48]
## Description of your Suggestion
I wanted to make an issue to have some discussion around what's going to happen with AudioBand on Windows 11. There's no more toolbar support as far as I can tell. I'm on W11 now, and I really miss this.  

## Alternative Solutions
- https://github.com/martinet101/ElevenClock I think these guys made a floating window that sits on top of the taskbar with a transparent background 
- #18

**Comment by svr333** [comment]
Im going to mention you seeing as you might not have received a notif
@zevisert

**Comment by zevisert** [comment]
Thanks for the response, but if the answer is to use ExplorerPatcher or StartAllBack I see this project as being dead. There's no fighting Microsoft, and eventually everyone will be on W11. If the only solution is then to ask everyone to revert their taskbar back to W10, then I'm not interested and I presume the same for most other AudioBand users. 

Sorry if that's bullish, that's just how I see it. I see your dono link, so I'll buy you a coffee for the great experience I had with this on W10. ✌️

**Comment by svr333** [comment]
Well that's currently, hopefully it will change in the future so it is no longer necessary to revert it back.
But I mostly work on this alone so time is very limited.

> Sorry if that's bullish, that's just how I see it. I see your dono link, so I'll buy you a coffee for the great experience I had with this on W10. ✌️

Greatly appreciate it, sorry for the inconvenience

**Comment by zEROma25** [comment]
> Well that's currently, hopefully it will change in the future so it is no longer necessary to revert it back.
> But I mostly work on this alone so time is very limited.

You can also check out this little app. Works great on Windows 11.
https://github.com/zhongyang219/TrafficMonitor

**Comment by RealLittlemrh** [comment]
Hey, I use explorer patcher normally, however when installing audioband my whole task bar just freezes and won't let me click anything. I can still update the taskbar's look if I had opened EP's settings before enabling audioband but can't click.

**Comment by TechSupport26** [comment]
Can you restart Explorer using Task Manager? That may fix your immediate issue.
When exactly did this happen? While the installation process was happening or after?

**Comment by RealLittlemrh** [comment]
it happens once I add audioband to the toolbar. I'll try restarting explorer though.

**Comment by svr333** [comment]
It sounds like an infinite loop or a silent crash, most likely from ExplorerPatcher because AudioBand will just shut down if it crashes.

Did you use this setup before and it worked?
If not, did you recently update Windows, ExplorerPatcher or AudioBand?

If nothing changed recently, please send the Audioband log file at `%temp%/AudioBand.log`

**Comment by RealLittlemrh** [comment]
[AudioBand.log](https://github.com/AudioBand/AudioBand/files/13465320/AudioBand.log)
I just found out about audioband and wanted to try it, here is my log though

**Comment by svr333** [comment]
This is the emptiest log I've ever seen, I think this is an issue where explorerpatcher freezes up when a new toolbar is created.
You can try opening the issue there or using an alternative like StartAllBack

**Comment by evnb** [comment]
I've tried both StartAllBack and Explorer Patcher. AudioBand only shows up for me when using EP in windows 10 mode. No options in StartAllBack made AudioBand show up. Unfortunately I want to keep my taskbar the way it is in windows 11, and just also have AudioBand. So looks like there's no solution for that atm. If i'm missing something please lmk!

**Comment by svr333** [comment]
Yeah Windows 11 had an update & explorerpatcher & startallback reacted accordingly

> Taskbar10: The Windows 10 taskbar option is now no longer available on 26002+. (#3053, e57a6b0)
This is to comply with Microsoft's removal of the stock Windows 10 taskbar in explorer.exe of said builds.

There is no way to use AudioBand on any recent Windows 11 build
